src/main/java/hudson/plugins/git/browser/GitRepositoryBrowser.java

```diff
@@ -117,5 +119,27 @@ public static URL encodeURL(URL url) throws IOException {
         }
     }
 
+    public static boolean initialChecksAndReturnOk(Item project, String cleanUrl){
+        if (cleanUrl == null) {
+            return true;
+        }
+        if (project == null || !project.hasPermission(Item.CONFIGURE)) {
+            return true;
+        }
+        if (cleanUrl.contains("$")) {
+            // set by variable, can't validate
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean checkURIFormat(String url, String browserName) throws URISyntaxException {
```

Comment:
I think you want this to only be used by classes that are extending GitRepositoryBrowser
```suggestion
    protected static boolean checkURIFormat(String url, String browserName) throws URISyntaxException {
```